src/modules/stream/types.js

```diff
@@ -44,8 +44,8 @@ function pipe(from, to, done) {
 }
 
 function getCommand(args) {
-    if (process.env.PROCESSING_PRIORITY && process.platform !== "win32") {
-        return ['nice', ['-n', process.env.PROCESSING_PRIORITY, ffmpeg, ...args]]
+    if (env.processingPriority != '' && process.platform !== "win32") {
```

Comment:
```suggestion
    if (env.processingPriority && process.platform !== "win32") {
```

Reply:
I did it to allow the user to specify number 0 in this variable, do we still remove it?

Reply:
in that case, i would probably check that `!isNaN(...)`

Reply:
sure, one moment